### DISCUSSION TITLE
nunjuck include not possible when using tpl from outside of project

### DISCUSSION THREAD

**Discussion by Acrontum-Carmichael** [j-d-carmichael/boats#66] [Ideas]
https://mozilla.github.io/nunjucks/templating.html#include

```
./src/components/jwt/access/model.yml.njk
```
Try to include this component from another project but it will throw an error.
```
{% include "../../../../ms-authentication/src/components/jwt/access/model.yml.njk" %}
```

It is possible to use the `$ref` from openapi/asyncapi - but this means juggling with other features eg allOf or nesting in another attribute. Would be cleaner to be able to just pull the code in.

**Comment by p-mcgowan** [comment]
might be able to extractthe code from the template path resolver and reuse it here - it probably just need to be converted to absolute path, since it ends up as-is in the build folder
